force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +47,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        if(age >= 18){
+            return true; 
+        } else {
+        return false; } // Replace null with the variable you used to store the result
```

Comment:
Think about that an expression using < or > will evaluate to. Can you solve this with one line of code and no if statements?